### ISSUE TITLE
Inaccessible color contrast in HTML input color when Dark mode

### ISSUE THREAD

**Issue by bramaudi** [kiwibrowser/android#192]
![Screenshot_Kiwi_Browser_20210825-140059.png](https://user-images.githubusercontent.com/14371119/130756273-b6561d48-d2d5-49ba-afa8-5fbdc27141c5.png)

As we can see (hope you can) the text color on "Set" and "Cancel" button is blend with the black background, need a while until i realize where that buttons are 😅
